### ISSUE TITLE
Feature Request: Tactile like hotkeys

### ISSUE THREAD

**Issue by mattdurham** [rxhanson/RectanglePro-Community#516]
I tend to use keyboard shortcuts more than mouse and would love behavior like [tactile](https://extensions.gnome.org/extension/4548/tactile/), where I hit a hot key to pull up my grid then each grid zone has a key.

So for a given grid below, when I hit the hotkey to trigger resize and only hit Q it takes up 1/2 by 1/4, but if I hit Q+S it takes up the left half of the screen.
```
| Q | W | E | R |
| A | S | D | F |
```

**Comment by rxhanson** [comment]
That's pretty clever, I like it. It's going to take some time before I'll be able to get to this, but will put it on my backlog. Thanks!

**Comment by radicarl** [comment]
I would find this very useful too. If you plan to implement this, couldyou add shortcuts to move the current window to next/previous space/display?

**Comment by cstmth** [comment]
Hey Ryan, is this still in the backlog/a priority?

I used Tactile extensively on Gnome because I own an ultrawide monitor and am missing it a lot. To get the same amount of flexibility with a 2x4 grid as above, one would currently need 22 unique key combinations in RectanglePro if I am not mistaken. With the proposed system, you only need 8 keys (Q, W, E, R, A, S, D, F) and every single combination in a 2x4 grid is possible.

**Comment by rxhanson** [comment]
Yes, this actually is still something I plan on implementing, but it's unfortunately still going to be a bit of time before I get to it. I'll post back here when I have something to try.
